vendor/mcmc/mcmc.h

```diff
@@ -1,14 +1,16 @@
 #ifndef MCMC_HEADER
 #define MCMC_HEADER
 
+#include <sys/uio.h>
+#include <stdint.h>
+
 #define MCMC_OK 0
 #define MCMC_ERR -1
 #define MCMC_NOT_CONNECTED 1
```

Comment:
Part of me wonders if it would be useful to separate out the connection codes from the protocol codes as well?
Without knowing the code all that much in depth yet, it might get tricky to track them as part of the same response code as the code grows, even if they're checked at different layers?

Reply:
Possibly, but with the read/write functions commented out nothing uses them outside of connect/disconnect. The parser uses WANT_READ but that's just communicated as a normal error code.

If/when the other functions are restored it'll be easier to reason about their return codes.

Reply:
Thinking more; potential return codes is kind of a function-by-function (for client facing API calls) thing anyway and if we're documenting things properly it should be at those levels. I think?

Reply:
> If/when the other functions are restored it'll be easier to reason about their return codes.
Ack

Reply:
> Thinking more; potential return codes is kind of a function-by-function (for client facing API calls) thing anyway and if we're documenting things properly it should be at those levels. I think?

Yea, as long as the contract is in an easily readable format, it may make the codebase easier to navigate. But that's just a nit.